--- configure
+++ configure
@@ -11640,8 +11640,8 @@
 		;;
 	*-*-darwin*)
 		DEBUG="-fno-common -g -Wall -fsigned-char"
-		CFLAGS="-fno-common -O4 -Wall -fsigned-char -ffast-math"
-		PROFILE="-fno-common -O4 -Wall -pg -g -fsigned-char -ffast-math"
+		CFLAGS="-fno-common -O3 -Wall -fsigned-char -ffast-math"
+		PROFILE="-fno-common -O3 -Wall -pg -g -fsigned-char -ffast-math"
 		;;
         *)
                 DEBUG="-g -Wall -fsigned-char"
