--- png.c.orig	2019-10-12 14:58:16.805296654 +0200
+++ png.c	2019-10-12 14:58:35.301963173 +0200
@@ -2388,8 +2388,8 @@
                       * discourage their use, skip the 'have_md5' warning below,
                       * which is made irrelevant by this error.
                       */
-                     png_chunk_report(png_ptr, "known incorrect sRGB profile",
-                         PNG_CHUNK_ERROR);
+                     /*png_chunk_report(png_ptr, "known incorrect sRGB profile",
+                         PNG_CHUNK_ERROR);*/
                   }
 
                   /* Warn that this being done; this isn't even an error since
