--- png.c
+++ png.c
@@ -2283,8 +2283,8 @@
                       * discourage their use, skip the 'have_md5' warning below,
                       * which is made irrelevant by this error.
                       */
-                     png_chunk_report(png_ptr, "known incorrect sRGB profile",
-                        PNG_CHUNK_ERROR);
+                     /*png_chunk_report(png_ptr, "known incorrect sRGB profile",
+                        PNG_CHUNK_ERROR);*/
                   }
 
                   /* Warn that this being done; this isn't even an error since
